server/galaxyls/services/completion.py

```diff
@@ -160,8 +161,8 @@ def get_auto_close_tag(self, context: XmlContext, trigger_character: str) -> Opt
         replace_range = None
         is_self_closing = trigger_character == "/"
         if is_self_closing:
-            start = Position(line=context.position.line, character=context.position.character)
-            end_character = context.position.character + 1
+            start = Position(line=context.position.line, character=context.position.character + 1)
```

Comment:
maybe its good to add a comment here, why the offset is needed

Reply:
Yeah, this code dealing with offsets is really messy. The +1 is due to the `context.position` been actually set on the `position_before_trigger` character on the previous call but we actually want to replace the trigger character `/` with the snippet `/>$0`.
Thanks for the feedback, I will try to refactor this a bit to see if I can simplify it or even remove the dirty offset math :)

Reply:
I added some comments to hopefully clarify this dance of character positions :)